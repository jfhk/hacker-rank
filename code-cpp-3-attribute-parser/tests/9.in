6 3
<t >
<tag3>
<tag2 name = "Name1">
</tag2>
</tag3>
</t>
t.tag3.tag2~name
t~name
t~value
