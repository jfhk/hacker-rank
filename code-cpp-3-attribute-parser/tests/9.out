Name1
Not Found!
Not Found!
